src/webserver/Static.jl

```diff
@@ -170,6 +170,7 @@ function http_router_for(session::ServerSession)
                 end
                 response
             else
+                println("A request to the Pluto server was made without providing a secret, here is the secret: $(session.secret)")
```

Comment:
Let's use `@info` here as we mostly do
```suggestion
                @info("A request to the Pluto server was made without providing a secret, here is the secret: $(session.secret)")
```